content/articles/code-style.md

```diff
@@ -0,0 +1,113 @@
+Title: Python Code Style at We Are Wizards
+Date: 2015-01-02
+Category: Code
+Authors: Gautier
+Short_summary: I write down some of the style habits I have when when writing Python code at We Are Wizards
+
+
+Here at We Are Wizards, we code review most of our commits for most of our
```

Comment:
I think you can remove "code" from "code" review and one "most".

"here at waw we review most of our commits for our projects"